sample/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//:pandoc.bzl", "pandoc", "PANDOC_FORMATS")
+
+# Conversion of README to various formats for testing.
+
+[pandoc(
+    name = "readme_" + fmt,
+    src = "//:README.md",
+    from_format = "markdown",
+    to_format = fmt,
+) for fmt in PANDOC_FORMATS]
+
+
```

Comment:
Could you please run `buildifier` on these files?

Reply:
Sure. Out of curiosiity, do you have a git presubmit hook for that? It seems buildifier alreadys returns 0.

Reply:
I don't have that set up right now. I usually just do a `%!buildifier` in Vim after making some edits. For a presubmit, we could always invoke `buildifier` and use `cmp` to compare changes against the input if we wanted one.

Reply:
This is how I do it https://github.com/jflex-de/jflex/blob/bbbd96445f66e6e5dfc880e27e60ede0b4497c0a/scripts/test-bzl-format.sh

Reply:
That's also one way to do it. Whatever works best for you :)